.github/workflows/packages.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       matrix:
         distro: 
         - centos-7
-        - debian-buster
+        - debian-bullseye
```

Comment:
I just want to highlight this good decision. We do not simply add new platforms to the packages action. We use a carefully selected set of platforms for this.